ddlambda.go

```diff
@@ -205,6 +208,10 @@ func (cfg *Config) toTraceConfig() trace.Config {
 		traceConfig.MergeXrayTraces, _ = strconv.ParseBool(os.Getenv(MergeXrayTracesEnvVar))
 	}
 
+	if !traceConfig.UniversalInstrumentation {
+		traceConfig.UniversalInstrumentation, _ = strconv.ParseBool(os.Getenv(UniversalInstrumentation))
```

Comment:
Nit pick. I know you're following prior art here, but it might be nice to debug log in the case that parsing fails (here and the two above too).

Without this, as a customer it would be really tough to figure out why my lambda instrumentation isn't working as I'd expected it.

Reply:
Holding off on this only because I feel a silent failure on parsing a `true/false` would hopefully be obvious to identify to the user! Also on the rare occasion somebody might do something like `DD_TRACE_ENABLED=blah` to disable tracing for a split second without removing the entire env var key for a confidence check/test, etc.